crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -171,9 +171,15 @@ impl DynamicScene {
     }
 
     // TODO: move to AssetSaver when it is implemented
-    /// Serialize this dynamic scene into rust object notation (ron).
+    /// Serialize this dynamic scene into the official Bevy scene format (`.scn` / `.scn.ron`).
+    ///
+    /// The Bevy scene format is based on [Rust Object Notation (RON)]. It describes the scene
+    /// in a human-friendly format. To deserialize the scene, use the [`SceneLoader`].
+    ///
+    /// [`SceneLoader`]: crate::SceneLoader
+    /// [Rust Object Notation (RON)]: https://crates.io/crates/ron
     #[cfg(feature = "serialize")]
-    pub fn serialize_ron(&self, registry: &TypeRegistryArc) -> Result<String, ron::Error> {
+    pub fn serialize(&self, registry: &TypeRegistry) -> Result<String, ron::Error> {
```

Comment:
Ideally this would support serializing the results to an arbitary output stream (i..e `std::io::Write`) instead of materializing the results to a string then writing that string out.

Reply:
This should probably done in a different PR, right? The output type change seems unrelated

Reply:
Yes, different PR